### PR TITLE
ENH move to latest upstream sha

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
Hmmmm

```
configure: error: Pthreads are required to build libatomic
```

**Comment by beckermr** [comment]
Error is in assembly?

```
2020-10-05T14:29:39.7546760Z configure:14894: arm64-apple-darwin20.0.0-gcc     -o conftest -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/gfortran_impl_osx-arm64-11.0.0.dev0 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -isystem /Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/include  -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs -Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -L$SRC_DIR/build_conda/arm64-apple-darwin20.0.0/libgcc -L/Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib -fno-sync-libcalls  -pthread -D_FORTIFY_SOURCE=2 -isystem $PREFIX/include -mmacosx-version-min=11.0 -O3 -isystem /Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/include  -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs -Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -L$SRC_DIR/build_conda/arm64-apple-darwin20.0.0/libgcc -L/Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs -Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -L$SRC_DIR/build_conda/arm64-apple-darwin20.0.0/libgcc -L/Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib conftest.c  >&5
2020-10-05T14:29:39.7550560Z <stdin>:22:15: error: index must be an integer in range [-256, 255].
2020-10-05T14:29:39.7551130Z         ldr     x4, [x4, ___stack_chk_guard];momd
2020-10-05T14:29:39.7551480Z                          ^
2020-10-05T14:29:39.7552260Z <stdin>:32:15: error: index must be an integer in range [-256, 255].
2020-10-05T14:29:39.7552800Z         ldr     x0, [x0, ___stack_chk_guard];momd
2020-10-05T14:29:39.7553230Z                          ^
2020-10-05T14:29:39.7553600Z configure:14894: $? = 1
2020-10-05T14:29:39.7554000Z configure: failed program was:
2020-10-05T14:29:39.7554280Z | /* confdefs.h */
2020-10-05T14:29:39.7554610Z | #define PACKAGE_NAME "GNU Atomic Library"
2020-10-05T14:29:39.7555020Z | #define PACKAGE_TARNAME "libatomic"
2020-10-05T14:29:39.7555410Z | #define PACKAGE_VERSION "1.0"
2020-10-05T14:29:39.7555830Z | #define PACKAGE_STRING "GNU Atomic Library 1.0"
2020-10-05T14:29:39.7556230Z | #define PACKAGE_BUGREPORT ""
2020-10-05T14:29:39.7556690Z | #define PACKAGE_URL "http://www.gnu.org/software/libatomic/"
2020-10-05T14:29:39.7557130Z | #define PACKAGE "libatomic"
2020-10-05T14:29:39.7557510Z | #define VERSION "1.0"
2020-10-05T14:29:39.7557870Z | #define STDC_HEADERS 1
2020-10-05T14:29:39.7558170Z | #define HAVE_SYS_TYPES_H 1
2020-10-05T14:29:39.7558550Z | #define HAVE_SYS_STAT_H 1
2020-10-05T14:29:39.7558910Z | #define HAVE_STDLIB_H 1
2020-10-05T14:29:39.7559270Z | #define HAVE_STRING_H 1
2020-10-05T14:29:39.7559620Z | #define HAVE_MEMORY_H 1
2020-10-05T14:29:39.7559980Z | #define HAVE_STRINGS_H 1
2020-10-05T14:29:39.7560350Z | #define HAVE_INTTYPES_H 1
2020-10-05T14:29:39.7560710Z | #define HAVE_STDINT_H 1
2020-10-05T14:29:39.7561070Z | #define HAVE_UNISTD_H 1
2020-10-05T14:29:39.7561420Z | #define HAVE_DLFCN_H 1
2020-10-05T14:29:39.7561790Z | #define LT_OBJDIR ".libs/"
2020-10-05T14:29:39.7562190Z | #define IFUNC_RESOLVER_ARGS uint64_t hwcap
2020-10-05T14:29:39.7562580Z | #define STDC_HEADERS 1
2020-10-05T14:29:39.7562890Z | #define STRING_WITH_STRINGS 1
2020-10-05T14:29:39.7563240Z | #define HAVE_FENV_H 1
2020-10-05T14:29:39.7563600Z | #define HAVE_INT1 1  
2020-10-05T14:29:39.7563940Z | #define HAVE_INT2 1  
2020-10-05T14:29:39.7564550Z | #define HAVE_INT4 1  
2020-10-05T14:29:39.7564910Z | #define HAVE_INT8 1  
2020-10-05T14:29:39.7565260Z | #define HAVE_INT16 1  
2020-10-05T14:29:39.7565640Z | #define HAVE_ATOMIC_LDST_1 1  
2020-10-05T14:29:39.7566010Z | #define HAVE_ATOMIC_LDST_2 1  
2020-10-05T14:29:39.7566410Z | #define HAVE_ATOMIC_LDST_4 1  
2020-10-05T14:29:39.7566790Z | #define HAVE_ATOMIC_LDST_8 1  
2020-10-05T14:29:39.7567160Z | #define HAVE_ATOMIC_LDST_16 0 
2020-10-05T14:29:39.7567560Z | #define HAVE_ATOMIC_TAS_1 1  
2020-10-05T14:29:39.7567930Z | #define HAVE_ATOMIC_TAS_2 1  
2020-10-05T14:29:39.7568320Z | #define HAVE_ATOMIC_TAS_4 1  
2020-10-05T14:29:39.7568690Z | #define HAVE_ATOMIC_TAS_8 1  
2020-10-05T14:29:39.7569080Z | #define HAVE_ATOMIC_TAS_16 1  
2020-10-05T14:29:39.7569460Z | #define HAVE_ATOMIC_EXCHANGE_1 1  
2020-10-05T14:29:39.7569920Z | #define HAVE_ATOMIC_EXCHANGE_2 1  
2020-10-05T14:29:39.7570290Z | #define HAVE_ATOMIC_EXCHANGE_4 1  
2020-10-05T14:29:39.7570950Z | #define HAVE_ATOMIC_EXCHANGE_8 1  
2020-10-05T14:29:39.7571290Z | #define HAVE_ATOMIC_EXCHANGE_16 0 
2020-10-05T14:29:39.7571670Z | #define HAVE_ATOMIC_CAS_1 1  
2020-10-05T14:29:39.7572040Z | #define HAVE_ATOMIC_CAS_2 1  
2020-10-05T14:29:39.7572430Z | #define HAVE_ATOMIC_CAS_4 1  
2020-10-05T14:29:39.7572790Z | #define HAVE_ATOMIC_CAS_8 1  
2020-10-05T14:29:39.7573180Z | #define HAVE_ATOMIC_CAS_16 0 
2020-10-05T14:29:39.7573560Z | #define HAVE_ATOMIC_FETCH_ADD_1 1  
2020-10-05T14:29:39.7573980Z | #define HAVE_ATOMIC_FETCH_ADD_2 1  
2020-10-05T14:29:39.7574370Z | #define HAVE_ATOMIC_FETCH_ADD_4 1  
2020-10-05T14:29:39.7574770Z | #define HAVE_ATOMIC_FETCH_ADD_8 1  
2020-10-05T14:29:39.7575190Z | #define HAVE_ATOMIC_FETCH_ADD_16 0 
2020-10-05T14:29:39.7575510Z | #define HAVE_ATOMIC_FETCH_OP_1 1  
2020-10-05T14:29:39.7575910Z | #define HAVE_ATOMIC_FETCH_OP_2 1  
2020-10-05T14:29:39.7576480Z | #define HAVE_ATOMIC_FETCH_OP_4 1  
2020-10-05T14:29:39.7576890Z | #define HAVE_ATOMIC_FETCH_OP_8 1  
2020-10-05T14:29:39.7577290Z | #define HAVE_ATOMIC_FETCH_OP_16 0 
2020-10-05T14:29:39.7577650Z | #define WORDSIZE 8
2020-10-05T14:29:39.7578010Z | /* end confdefs.h.  */
2020-10-05T14:29:39.7578360Z | #include <pthread.h>
2020-10-05T14:29:39.7578700Z |       void *g(void *d) { return NULL; }
2020-10-05T14:29:39.7579050Z | int
2020-10-05T14:29:39.7579270Z | main ()
2020-10-05T14:29:39.7579550Z | {
2020-10-05T14:29:39.7579900Z | pthread_t t; pthread_create(&t,NULL,g,NULL);
2020-10-05T14:29:39.7580280Z |   ;
2020-10-05T14:29:39.7580580Z |   return 0;
2020-10-05T14:29:39.7580870Z | }
```

**Comment by beckermr** [comment]
I wonder if `-fstack-protector-strong` is related?

**Comment by beckermr** [comment]
@isuruf if you have a moment to look at this, any guidance would be appreciated!

**Comment by beckermr** [comment]
thanks @isuruf! I was about to put those flags at the top of the file!

**Comment by beckermr** [comment]
These are similar segfaults to what I was seeing with the build of the old sha. I wonder if they are related to cctools and not the compiler.

**Comment by isuruf** [comment]
Can you send a PR to comment out https://github.com/tpoechtrager/cctools-port/blob/master/cctools/misc/strip.c#L789 and https://github.com/tpoechtrager/cctools-port/blob/master/cctools/misc/strip.c#L730

**Comment by beckermr** [comment]
done

**Comment by isuruf** [comment]
https://github.com/conda-forge/cctools-and-ld64-feedstock/pull/23 too if you have time

**Comment by beckermr** [comment]
I added a patch. Can you push to the rc branch assuming it passes?

**Comment by beckermr** [comment]
@conda-forge-admin restart ci

**Comment by beckermr** [comment]
yaaaay the fix to cctools worked! 

@isuruf ready for final review and merge!

File: recipe/build.sh [hunk 1]
```diff
@@ -139,8 +147,8 @@ if [[ "$host_platform" == "$target_platform" ]]; then
     sed -i.bak "s/USE_FORTRAN_FALSE=.*/USE_FORTRAN_FALSE='#'/g" $SRC_DIR/libgomp/configure
     sed -i.bak "s/USE_FORTRAN_TRUE=.*/USE_FORTRAN_TRUE=/g" $SRC_DIR/libgomp/configure
   fi
-
-  make -j"${CPU_COUNT}" || (cat $TARGET/libgomp/*.log && false)
+  
+  make -j"${CPU_COUNT}" || (cat $TARGET/libatomic/*.log && false)
```
Comment (isuruf) [inline recipe/build.sh]:
```suggestion

  make -j"${CPU_COUNT}" || (cat $TARGET/libquadmath/*.log && false)
```

**Comment by beckermr** [comment]
thanks @isuruf !

**Pull request merged**